docker-compose.yml

```diff
@@ -2,183 +2,236 @@ version: "3.8"
 services:
   stride1:
     image: stridezone:stride
+    user: root
     volumes:
       - ./scripts/state/stride1:/home/stride/.stride
+    command: strided start --home /home/stride/.stride
```

Comment:
I don't think these commands are necessary since they're in the dockerfile

Reply:
in dockerfile, home is pointed to /root/.stride by default so the docker fails. Best would be to change this in Dockerfile CMD.